src/agentversion

```diff
@@ -1 +1 @@
-2.155.0
\ No newline at end of file
+2.156.0
```

Comment:
this will be 157 unless you are planning to hotfix to 156

Reply:
Okay, Didn't know it matches the Sprint number. So just increased it by 1.